bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -244,6 +244,22 @@ spec:
               - -c
               - |
                 set -x
+                # Upgrade the db if required.
```

Comment:
I can't find anything specifically wrong in moving this out of the loop and providing the freedom of doing an upgrade on any control-plane node. Having said that I'd be curious to see why this code was restricted to the initiator in the first place, maybe @numansiddique (https://github.com/openshift/cluster-network-operator/pull/755/files#diff-58be2f354a42fe685cf7eadff055a78e1f9be499adf74f786cde1a26d509cc7bR155) can say why it used to be done only for `MASTERIP` which later became `INITIATOR`?

Reply:
Spoke with Numan and he said there was no real reason only the initiator was originally doing it